report/sections/Chapter 02 - Preliminaries/preliminaries.tex

```diff
@@ -154,4 +154,6 @@ \section*{BenchmarkDotnet}
 
 BenchmarkDotnet\cite{Benchmarkdotnet} is a benchmarking framework allowing time measurements of operations performed in dotnet runtimes. 
```

Comment:
```suggestion
BenchmarkDotNet\cite{Benchmarkdotnet} is a benchmarking framework for measuring the speed of operations performed in .NET runtimes. 
```

Reply:
Kinda implies that it benchmarks operations in the .NET runtime, not using .NET to benchmark operations?